app/controllers/sciruby_github_controller.rb

```diff
@@ -6,12 +6,12 @@ class ScirubyGithubController < ApplicationController
   def repos
     # getting data from url : https://api.github.com/orgs/Sciruby/repos
     # An example: http://nbviewer.jupyter.org/github/athityakumar/daru-io/blob/iruby-examples/iruby/json_importer.ipynb
-    @df_repos = Daru::DataFrame.from_json('https://api.github.com/orgs/Sciruby/repos', Repo_name: '$..name', forks: '$..forks', open_issues: '$..open_issues_count')
-
-    # Actually no need of this respond_to block right now.
-    respond_to do |format|
-      format.html  # just run the sciruby_github/repos.erb
-    end
+    @df_repos = Daru::DataFrame.read_json(
+      'https://api.github.com/orgs/Sciruby/repos',
+      Repo_name: '$..name',
+      forks: '$..forks',
+      open_issues: '$..open_issues_count'
```

Comment:
How about keys consistency? E.g. `repo_name` (snake_case for all, or CamelCase for all)